metaworld/envs/mujoco/sawyer_xyz/sawyer_bin_picking.py

```diff
@@ -47,8 +47,8 @@ def __init__(self):
         )
 
         self.goal_space = Box(goal_low, goal_high)
-        self.obj_and_goal_space = Box(low=np.array([-0.22, -0.02]),
-                                      high=np.array([0.6, 0.8]))
+        self.obj_and_goal_space = Box(low=np.array([-0.21, 0.65]),
```

Comment:
where do these magic number come from? can they get some named constants instead?

Reply:
@avnishn thoughts? These changes are from your branch

Reply:
I looked into it and cleaned up the magic numbers